pages/blueprint/index.js

```diff
@@ -407,75 +464,14 @@ class BlueprintPage extends React.Component {
-                                    )}
-                                  </td>
-                                  <td data-td="password">{user.password && <span className="fa fa-check" />}</td>
-                                  <td data-td="sshkey">
-                                    {user.key !== undefined && (
-                                      <span>
-                                        <span className="fa fa-check" />
-                                        {` `}
-                                        {user.key.split(" ")[2] || ""}
-                                      </span>
-                                    )}
-                                  </td>
-                                  <td className="cmpsr-table-actions">
-                                    <button
-                                      className="btn btn-default"
-                                      type="button"
-                                      aria-label={`${formatMessage(messages.userEdit)} ${user.name}`}
-                                      onClick={e => this.handleShowModalEditUser(e, user)}
-                                      data-btn="edit"
-                                    >
-                                      <span className="pficon pficon-edit" />
-                                    </button>
-
-                                    <div className="dropdown btn-group dropdown-kebab-pf">
-                                      <button
-                                        aria-label={`${formatMessage(messages.userKebab)} ${user.name}`}
-                                        className="btn btn-link dropdown-toggle"
-                                        type="button"
-                                        data-btn="more"
-                                        id="dropdownKebab"
-                                        data-toggle="dropdown"
-                                        aria-haspopup="true"
-                                        aria-expanded="false"
-                                      >
-                                        <span className="fa fa-ellipsis-v" />
-                                      </button>
-                                      <ul className="dropdown-menu dropdown-menu-right" aria-labelledby="dropdownKebab">
-                                        <li>
-                                          <a href="#" onClick={e => this.handleDeleteUser(user.name, e)}>
-                                            {formatMessage(messages.userDelete)}
-                                          </a>
-                                        </li>
-                                      </ul>
-                                    </div>
-                                  </td>
-                                </tr>
-                              ))}
-                            </tbody>
-                          </table>
+                        <div className="user-list">
```

Comment:
Can `.user-list` be moved further up the DOM to the parent `div.form-group`? This would allow us to add a few more styles to create better vertical alignment among the label "Users" and the table headers. For example:
```
.user-list .control-label {
  padding-top: .375rem;
}
.user-list th {
  padding-top: .375rem;
} /* this one might need more specificity 🤷 */
```

The `.375rem` is based on the padding-top currently applied to the "Create User Account" button. Also note, I would consider these styles as a temporary fix, until the rest of the page is refactored.